docs/paper/admin/reference/system-properties.md

```diff
@@ -0,0 +1,178 @@
+---
+slug: /reference/system-properties
+---
+
+# Paper System Properties 
+
+These system properties can be set when you start your server allowing for the configuration of various settings.
+
+:::danger Danger Ahead
+
+Setting flags for the JVM can alter how it operates and the same goes for the Paper server.
+If you are unsure about what a flag does, it is recommended that you **do not use it**.
+
+:::
+
+## How they work
+
+System properties are set when you start your server. For example, if you are using a `.bat` or a `.sh` file to start your server, you can add the system properties to the file. For example:
+
+```bash
+java -Dpaper.log-level=FINE -jar paper.jar
+```
+
+Where a `-D` is used to set a system property, and the system property is `paper.log-level` with a value of `FINE`. Otherwise, just add them to the start command
+
+## List of System Properties
+
+#### paper.playerconnection.keepalive:
+
+- **default**: `30`
+- **description**: Controls how long the player connection will wait before closing when not receiving any keepalives, in seconds.
+
+#### timings.bypassMax:
+
+- **default**: `unset`
```

Comment:
For all of these unset ones - some of them need to be explicitly set to `true` and others just check for non-null. Maybe make note or standardize this? I'm not sure that it really matters all that much, though.

Reply:
Yeah, It wouldn't be too hard to document but most I saw just used `!= null` checks... lmk if this is wanted

Reply:
taking a second look should really be fine to just say set it to `true` for everything - as that does work regardless.